src/site/xhtml/index.xhtml.vm

```diff
@@ -306,5 +306,20 @@
       In the above example, two environments are running when the incremental update command is called.
     </div>
   </p>
+
+  <h2 id="google_analytics_tracking">Google Analytics tracking</h2>
+  <p>
+    The eosgi-maven-plugin collects anonym usage statistics. The collected statistics are used to get budget fo further development.
+    Collected data are:
+  </p>
+  <ul>
+    <li>analytics referer:  the eosgi-maven-plugin or other plugin. Example eclipse-e4-plugin, which is under development now.</li>
+    <li>executed goal: the executed goal.</li>
+    <li>mac address hash: the MAC address hash to connect usage to anonym user. SHA-1 algorithm is used to hash the mac address.</li>
```

Comment:
"the hashed value of the MAC address of the first network interface controller of the user's computer. SHA-1 algorithm is used ti create the hash to make the user anonym and unique."